notebooks/3_KernelMethods.ipynb

```diff
@@ -717,10 +719,11 @@
     "\n",
     "regularization = 1e-12\n",
     "\n",
-    "krr = KRR(regularization=regularization, kernel_type=kernel_type)\n",
+    "krr = KernelRidge(alpha=regularization, kernel = 'rbf', gamma = 1.0)\n",
     "krr.fit(X_train, Y_train)\n",
-    "Yhat_train = krr.transform(X_train).reshape(-1,Y.shape[-1])\n",
-    "PKY = krr.PKY\n",
+    "Yhat_train = krr.predict(X_train).reshape(-1,Y.shape[-1])\n",
+    "PKY = np.linalg.solve(K_train + regularization*np.eye(n_train), \n",
```

Comment:
```suggestion
    "PKY = krr.dual_coef_
```

Reply:
I've tried to use `PKY = krr.dual_coef_`. But I found This would lead to errors.
![image](https://user-images.githubusercontent.com/56018055/99545435-a8a2e800-29b5-11eb-85ad-a437adab764e.png)
![image](https://user-images.githubusercontent.com/56018055/99545550-c96b3d80-29b5-11eb-9390-a5d5bdab6c67.png)
However, there will be no problem if we explicitly calculate `PKY`.
![image](https://user-images.githubusercontent.com/56018055/99545956-2d8e0180-29b6-11eb-9ec1-b62e5fa060a2.png)
![image](https://user-images.githubusercontent.com/56018055/99545994-3da5e100-29b6-11eb-92c6-0d56fc136604.png)

Reply:
Please look into this and solve it. _dual_coef _should_ give the projector, but there is *always* a chance of it being rotated with respect to the other projector. Please solve this yourself.

Reply:
Done. The problem is that KernelRidge won't automatically center the kernel matrix